htdocs/install/mysql/data/llx_c_type_contact.sql

```diff
@@ -43,6 +43,7 @@ insert into llx_c_type_contact(rowid, element, source, code, libelle, active ) v
 insert into llx_c_type_contact(rowid, element, source, code, libelle, active ) values (60, 'facture', 'external', 'BILLING',       'Contact client facturation', 1);
 insert into llx_c_type_contact(rowid, element, source, code, libelle, active ) values (61, 'facture', 'external', 'SHIPPING',      'Contact client livraison', 1);
 insert into llx_c_type_contact(rowid, element, source, code, libelle, active ) values (62, 'facture', 'external', 'SERVICE',       'Contact client prestation', 1);
+insert into llx_c_type_contact(rowid, element, source, code, libelle, active ) values (63, 'facture', 'external', 'CUSTOMER',      'Contact client suivi facturation', 1)
```

Comment:
This contact "Contact client suivi facturation" already exists with id 60 and key 'BILLING'.

Reply:
@eldy 
if you create a shipping with external contact "CONTACT", when you create an invoice from this shipping you have an error if "CONTACT" type not exists for invoice in "element_contact"... maybe it's fixed in version 6 or higher?